CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## qiskit-aqt-provider v1.0.0
+
+Releases in the v1.x.x series support the AQT quantum computing portal ARNICA at https://arnica.aqt.eu.
```

Comment:
```suggestion
Releases in the v1.x.x series support the AQT quantum computing portal ARNICA at https://arnica.aqt.eu/api/v1.
```
https://arnica.aqt.eu redirects to https://www.aqt.eu/qc-systems so maybe better mention the API